frontend/src/v5/ui/controls/formNumberField/formNumberField.component.tsx

```diff
@@ -15,9 +15,6 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import styled from 'styled-components';
 import { FormTextField } from '@controls/formTextField/formTextField.component';
 
-export const FormNumberField = styled(FormTextField).attrs({ type: 'number' })`
-	width: 50%;
-`;
+export const FormNumberField = (props) => (<FormTextField type="number" {...props} />);
```

Comment:
This component was created to be 50% of the normal textfield. In practice, however, this component always uses 100% of the space provided (be that in a flex,grid, or block container)
This is where it used as the moment
![image](https://user-images.githubusercontent.com/43547703/197211698-d32cf13f-9f63-4f83-addf-94da35580014.png)

Similarly (but it is not being used there yet), should we replace these fields in `container settings`, it would still use 100% of the width 
![image](https://user-images.githubusercontent.com/43547703/197211547-43440488-3054-4ee8-b6ef-ee7c2e08caf0.png)